internal/sail/client/client_test.go

```diff
@@ -39,12 +40,29 @@ func TestBroadcast(t *testing.T) {
 	assert.Equal(t, 2, len(f.conn().json.(webview.View).Resources))
 }
 
+func TestSailRoomConnectedAction(t *testing.T) {
+	f := newFixture(t)
+	defer f.TearDown()
+	go f.store.Loop(f.ctx)
+	f.client.OnChange(f.ctx, f.store)
+
+	time.Sleep(10 * time.Millisecond)
```

Comment:
are you looking for the waitForActions helper function? (getActions is kind of an idiosyncractic api that generally isn't called directly)

Reply:
ooooh! yes, i am!

Reply:
I'm leaving that for another PR though (see #1540 for refactor to make it available)

Reply:
nvm, just did!